src/System.Net.Requests/src/System/Net/WebException.cs

```diff
@@ -65,6 +70,16 @@ public WebResponse Response
             }
         }
 
+        void ISerializable.GetObjectData(SerializationInfo serializationInfo, StreamingContext streamingContext)
+        {
+            GetObjectData(serializationInfo, streamingContext);
+        }
```

Comment:
Do you know why it's not serializing at least ```_status```?  I see that code is commented out:
https://referencesource.microsoft.com/#System/net/System/Net/WebException.cs,163
It'd be interesting to know why.  Can you see anything in the history of the code in TFS to indicate when and why that happened?